client/src/styles.scss

```diff
@@ -16,6 +16,7 @@ $my-primary: mat.define-palette(mat.$indigo-palette, 500);
   $primary: map-get($theme, primary);
   $accent: map-get($theme, accent);
   $warn: map-get($theme, warn);
+  $activeTabHighlight: white;
```

Comment:
Can we get rid of this color value? It seems like a hold over from the old code-base, and the color choice doesn't match with the rest of the current project.

Reply:
Done